src/lib/routes/backstage.ts

```diff
@@ -16,6 +17,14 @@ class BackstageController extends Controller {
                 res.end(await prometheusRegister.metrics());
             });
         }
+
+        if (config.server.enableHeapSnapshotEnpoint) {
+            this.get('/heapSnapshot', async (req, res) => {
```

Comment:
Maybe better to do kebab-case?

```suggestion
            this.get('/heap-snapshot', async (req, res) => {
```